src/index.html

```diff
@@ -4,9 +4,332 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__ltem">
+            <a href="#Apple"
+              class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__ltem">
+            <a href="#Samsung"
+              class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__ltem">
+            <a href="#Smartphones"
+              class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__ltem">
+            <a
+              href="#Laptops & Computers"
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__ltem">
+            <a href="#Gadgets"
+              class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__ltem">
+            <a href="#Tablets"
+              class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__ltem">
+            <a href="#Photo"
+              class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Video"
+              class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="cards">
+        <div data-qa="card" class="card cards__card">
+          <img class="card__image" src="/src/images/imac.jpeg"
+          alt="iMac A1419">
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__review-count">Reviews: 5</p>
+          </div>
+
+          <div class="card__price-container">
+            <h3 class="card__price-label">Price:</h3>
+            <p class="card__price">$2,199</p>
+          </div>
+
+          <a class="card__buy" href="#"
+          data-qa="card-hover">
+            Buy
+          </a>
```

Comment:
The same as above, need to check and fix it everywhere
```suggestion
          <a
            class="card__buy"
            href="#"
            data-qa="card-hover"
          >
            Buy
          </a>
```